### PR TITLE
Adding basic support for drop_duplicates/DISTINCT queries

### LINKED ISSUE #12: Support unique/DISTINCT command
For many common queries, we must support the command `pandas.unique()`. We will map this to `DISTINCT` in SQL.

### PR DESCRIPTION
**Pull request by arjunrawal4**
Supporting `drop_duplicates()`. Limitations - does not allow column subsets due to SQL query structure and does not currently support any advanced options. 

Closes #12

### CODE REVIEW

File: pandasql/core.py [hunk 1]
```diff
@@ -469,6 +469,18 @@ def groupby(self, by, as_index=False):
         """TODO: support other pandas groupby arguments"""
         return GroupByDataFrame(self, by=by, as_index=as_index)
 
+    def drop_duplicates(self, subset=None, keep='first', inplace=False,
+                        ignore_index=False):
+        if subset is not None:
+            raise ValueError('Subset support has not been added')
+        if inplace is True:
```
Comment (rohankumar42) [inline pandasql/core.py]: Nit: remove `is True` (also below)

File: pandasql/core.py [hunk 2]
```diff
@@ -593,12 +605,17 @@ def __init__(self, source: DataFrame, col, name=None):
...
-        self._sql_query = 'SELECT {} FROM {}'.format(', '.join(self.columns),
+        self.dedup = drop_duplicates
+        self.columns = source.columns[source.columns.isin(cols)]
+        dist = "DISTINCT " if self.dedup is True else ""
+        self._sql_query = 'SELECT {}{} FROM {}'.format(dist,
+                                                     ', '.join(self.columns),
                                                      self.sources[0].name)
 
     def _pandas(self):
+        if self.dedup is True:
```
Comment (rohankumar42) [inline pandasql/core.py]: Nit: same as above

File: tests/test_data_frame.py [hunk 3]
```diff
@@ -603,5 +603,23 @@ def test_complex_write_attr_access(self):
         assertDataFrameEqualsPandas(df.p, pd.DataFrame(base_df.p))
         assertDataFrameEqualsPandas(df_1, pd.DataFrame(base_df_1))
 
+    def test_drop_duplicates(self):
```
Comment (rohankumar42) [inline tests/test_data_frame.py:635]: This only tests `drop_duplicates` on SQLite. Let's add corresponding tests in `test_pandas_execution.py` too.

Also, maybe check `df.sql()` in line with other tests in this file?

**Review by rohankumar42 — CHANGES_REQUESTED** [review]

File: tests/test_data_frame.py (hunk 3 above)
Comment (arjunrawal4) [reply tests/test_data_frame.py:635]: I'll add the same tests to `test_pandas_execution.py`, but just for future simplicity is there a reason to have them be separate files? Could we just have the test suite run once with offloading and once without?

Comment (rohankumar42) [reply tests/test_data_frame.py:635]: In retrospect, it was probably a poor test design choice. We should do that. I'll make an issue!

**Review by rohankumar42 — APPROVED** [review]

**Pull request merged**